First Capstone Project

BenM4k opened the pull request:
In this branch, I implemented: 

- the HTML structure for the website, which means the home and about pages.
- all the style and made the site responsive
- the navbar button for mobile and a dynamic featured speaker section using JavaScript

BenM4k committed:
major update

html: navbar and all the main sections

css: all syles for the navbar and the sections using scss

javascript: navbar interaction

BenM4k committed:
update: responsiveness

I refactored the code so it can fit on mobiles, tablets, pcs and larger screens

BenM4k committed:
fix: linter errors

fixed error for both eslint and stylelint

BenM4k committed:
update: featured speakers

made the section dynamic using js

BenM4k committed:
update: readme file

In this commit I designed the readme file for my project

BenM4k committed:
fix: images alt errors

Added alts for all my images

BenM4k committed:
Update README.md

updated getting started section

osamaashraf6 commented on index.html:
```diff
@@ -5,37 +5,94 @@
 </head>
 <body>
     <header>
         <div class="menu-btn">
             <span class="burger"></span>
         </div>
         <nav class="navbar">
+            <a href="index.html" class="logo"><img src="assets/1280px-Daum_Kakao_wordmark_-_2014.svg.png" alt="logo_img"></a>
             <ul class="menu-navbar">
                 <li class="menu-navbar__item">
-                    <a href="/webpages/about.html">about</a>
```
- [ ] You did a great job 😃 on the project, but I see that you forgot to add the content of the sharing links before the nav links and the logo, so please try to add the content of Social links before the header of the nav links to make it like the Behance Design 👌💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/233058223-9d65de6d-e031-42ed-9e75-e0d047ffc962.png)   |   ![image](https://user-images.githubusercontent.com/83360636/233058263-00459458-0d8a-41bc-82f4-ae299099c4b7.png)  |

osamaashraf6 replied:
- [ ] It will be better 😊 if you put the navigation links in a suitable position on the mobile version for example you can put them at the center to make it nice for the user 💯
- [ ] Also consider increasing the size of the link to be great for the users and to be like the Behnace design 😃
- [ ] Also consider decreasing the margin between the ```X``` icon and the links to be nice for the users 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
 | --- |
|![image](https://user-images.githubusercontent.com/83360636/233070233-6f871d18-da8e-45c4-9821-63e0c3e69356.png)   |

osamaashraf6 commented on README.md:
```diff
@@ -1 +1,173 @@
+I would like to thank Microverse and all My Micronauts partners for their help on this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
- [x] While you make a great effort 😃 I see that you forgot to add the acknowledgment for the author of the original design, so please 😊 The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).🚀💯

osamaashraf6 commented on index.html:
```diff
@@ -5,37 +5,94 @@
             <ul class="menu-navbar">
                 <li class="menu-navbar__item">
-                    <a href="/webpages/about.html">about</a>
+                    <a href="pages/about.html" class="active">About</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/program.html">Program</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/join.html">Join</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/sponsor.html">Sponsor</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/news.html">News</a>
                 </li>
             </ul>
         </nav>
     </header>
     <main>
         <section class="home">
```
- [ ] While you make a great effort 😃 I see that the content of the page is not aligning exactly like the Behance I think the content needs to move to the left a little to be nice, so I think 😊it will be better if you put the content of the page in a container div and give it the appropriate width to align all the content of the page like the Behance Design 💯👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/233062561-74a9d2d9-cd34-4468-999e-394c83dbb1d7.png)  |    ![image](https://user-images.githubusercontent.com/83360636/233062451-e7a83321-e04f-47a6-9186-fd8f8cb483d8.png) |

osamaashraf6 commented on index.html:
```diff
@@ -5,37 +5,94 @@
+            <a href="index.html" class="logo"><img src="assets/1280px-Daum_Kakao_wordmark_-_2014.svg.png" alt="logo_img"></a>
             <ul class="menu-navbar">
                 <li class="menu-navbar__item">
-                    <a href="/webpages/about.html">about</a>
+                    <a href="pages/about.html" class="active">About</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/program.html">Program</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/join.html">Join</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/sponsor.html">Sponsor</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/news.html">News</a>
                 </li>
             </ul>
         </nav>
     </header>
     <main>
         <section class="home">
-
+            <h2 class="home-title">"Hello! Sharing world"</h2>
+            <h1 class="home-title__big">Creative commons global summit 2015</h1>
+            <p class="home-paragraph"> Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sunt autem, delectus sit animi nam nihil, voluptatibus nobis natus atque quo qui blanditiis. Nobis ad autem nemo at reprehenderit ipsa blanditiis voluptas non nulla consequatur itaque voluptatibus.</p>
+            <h2 class="home-footer">2015.10.15(THU) ~ 16(FRI)</h2>
+            <h3 class="home-copywrite">&copy; National Museum of Korea, Art Center Nabi and more</h3>
         </section>
```
- [x] Kindly 😃 try to add a background image like the Behance you can use any background image, but please 💯try to use the suitable one to be like the Behance and light for the page 😊👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/233063329-7cfa899d-524b-4485-8e3a-b4a759a1fae1.png)   |   ![image](https://user-images.githubusercontent.com/83360636/233063195-a3ae3132-9a18-4872-a445-165d739b10f9.png)  |

osamaashraf6 commented on index.html:
```diff
@@ -5,37 +5,94 @@
+                        <img src="/assets/icon4.svg" alt="list_image_four" class="list-head__img">
+                        <h3>Workshop</h3>
+                    </div>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
```
- [ ] It will be better 😃 if you try to stick to the Behance Design to make the project nice like the Behance, so I see here that the cards of the main program section need some padding from left and right to be like the Behance, so please 😊 try to increase the padding of the card to make it nice like the Behance 🚀👌
- [ ] Also please try to decrease the padding from the top and bottom to make it like the Behance Design 🚀
- [ ] Also please 💯 consider decreasing the margin between the cards as you see here it is bigger little than the Behance, so please try to make it like the Behnace design 😊
- [ ] Also It will be better if you add the hover effect to the cards to give the user a beautiful sense when Using the website 🚀👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/233064851-06aa4d81-56cf-4312-8835-bdc7e9782cdb.png)   |  ![image](https://user-images.githubusercontent.com/83360636/233064802-8a741cb2-f0dd-4386-995f-62f3931a4d91.png)  |

osamaashraf6 commented on index.html:
```diff
@@ -5,37 +5,94 @@
+            </ul>
+            <button class="main-program__button">Join CC Global Summit 2015</button>
         </section>
         <section class="featured-speakers">
```
- [x] While you make a great effort 😃 I see that the link here is not like the Behance, so it will be better 😊if you give it the style like the Behance to make it nice 🚀👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/233066163-dd72c916-8cb1-492c-8fd1-b975644f20ad.png)   |    ![image](https://user-images.githubusercontent.com/83360636/233066087-4cb22d60-a01c-496d-bf0c-8d27ce8a5270.png)  |

osamaashraf6 commented on index.html:
```diff
@@ -5,37 +5,94 @@
+                        <img src="/assets/forum.svg" alt="list_image_three" class="list-head__img">
+                        <h3>Forum</h3>
+                    </div>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium cum cumque quasi</p>
```
- [x] I think it will be better 😃 if you increase the line height of the text inside the card as you see here it is not like the Behance and its appearance not comfortable for the eye, so let's 💯 try to increase it to make it like the Behance 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/233066922-824d6369-83ea-47ec-91d3-7f122f07d742.png)  |    ![image](https://user-images.githubusercontent.com/83360636/233066869-2d653f34-c9b3-41fa-b8ee-ae2f54fa0fec.png)  |

osamaashraf6 commented on index.html:
```diff
@@ -5,37 +5,94 @@
+            <button class="featured-speakers__button">More</button>
         </section>
     </main>
     <footer>
 
     </footer>
```
- [x] While you make a great effort 😃 I see that you forgot to add the footer at the bottom of the page, so please try to add the content of the footer to make the design nice and stick to the Behance design 💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance  design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/233068159-e26e5840-88cb-441f-a9b1-9ba1a41b0cee.png)   |  ![image](https://user-images.githubusercontent.com/83360636/233068305-3f21a2ed-91cb-4537-b57a-c140d2a89c98.png)  |

osamaashraf6 commented on index.html:
```diff
@@ -5,37 +5,94 @@
-
-        </section>
-        <section class="partners">
-            
+            <h2 class="featured-speakers__title">Featured Speakers</h2>
+            <span class="featured-speakers__line"></span>
+            <ul class="featured-speakers__container"></ul>
+            <button class="featured-speakers__button">More</button>
         </section>
```
- [x] While you make a great effort 😃 I see that you forgot to add the Partners content at the section of the partner, so please try to add the content of the Partners to make the design nice and stick to the Behance design 💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/233068861-3cba43e5-fcf0-40f1-ad81-f15f568761b9.png)   |    ![image](https://user-images.githubusercontent.com/83360636/233068940-96948c89-53bb-452d-857a-3ae3489967a9.png) |

osamaashraf6 reviewed (changes requested):
Hi @BenM4k 😊, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![SoCloseAlmostGIF (2)](https://user-images.githubusercontent.com/83360636/233061197-d7b2386d-b19b-45cf-813b-bcf87f8475ba.gif)

## Highlights🎉🎉

- [x] Not linter errors present
- [x] Good descriptive pull request
- [x] Nice design 💯 

## Required Changes ♻️

- [x] Kindly 😃try to add the Project documentation, I think you forgot to add the Video presentation, so please 😊 try to add the video presentation for the project following these steps in your mind to make it nice and learn from yourself how to be confident when making a presentation 😊⤵️

> If you can not see the image clearly just click on it to open a new tab in the browser😊

|Project documentation| 
| --- |
|   ![image](https://user-images.githubusercontent.com/83360636/233060688-411443fa-ddbc-4f4d-ba66-ee78241ef437.png)  |

- [x] While you make a great effort I see that you put the link to the live version of the portfolio project, but here we are at the capstone conference project, so please try to add the correct live version link using the GitHub pages 😊⤵️

| Project requirements |
| ---|   
| ![image](https://user-images.githubusercontent.com/83360636/233073203-9eac164a-60e9-4fc0-bc21-9306a424520c.png)   |

- Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenM4k committed:
fix: design and readme

Added credits to the design owner in the readme file

fix every changes requested by the reviewer

BenM4k committed:
fix: stylelint errors

assigned classes to avoid conflicts

BenM4k committed:
update: readme and index.html

added the social media navbar, fixed the linters error and the remaining styles errors

added the video presentation of the project

AdnanOlarmmi commented on README.md:
```diff
@@ -1 +1,176 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```
- [x] Like the previous code reviewer suggested, kindly deploy the app and add the appropriate `live-demo` link.


This is where the current live demo link is redirecting to. 😉 
<img width="161" alt="image" src="https://user-images.githubusercontent.com/41163571/233260692-e3527adb-4584-4758-b6f0-7e912d54ddbb.png">

Then the about  link leads to a 404 Github error page
<img width="645" alt="image" src="https://user-images.githubusercontent.com/41163571/233260997-deecf8dd-aa18-4ada-86c3-5eafe363ff2d.png">

BenM4k replied:
To deploy it I have to merge the branch. To merge the branch You have to finish with the corrections.

AdnanOlarmmi replied:
Hi @BenM4k 😊 ... You can deploy a branch, you don't necessarily have to deploy the main branch... Contact me if you need more clarifications.

AdnanOlarmmi commented on README.md:
```diff
@@ -1 +1,176 @@
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
```
- [OPTIONAL] Since you built this project solely by yourself, I think you should correct this to `[👥 Author](#author)`, without the "s", and change it also in the author content header, as in `## 👥 Author <a name="author"></a>`

AdnanOlarmmi commented on index.html:
```diff
@@ -5,37 +5,117 @@
+            <h2 class="main-program__title">Main Program</h2>
+            <span class="main-program__line"></span>
+            <ul class="main-program__container">
+                <li class="main-program__list">
+                    <div class="list-head">
+                        <img src="/assets/lecture.svg" alt="list_image_one" class="list-head__img">
+                        <h3>Lecture</h3>
+                    </div>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, nisi esse.</p>
+                </li>
+                <li class="main-program__list">
+                    <div class="list-head">
+                        <img src="/assets/icon2.svg" alt="list_image_two" class="list-head__img">
+                        <h3>CC Exhibition</h3>
+                    </div>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. </p>
```
- [x] Nice one fixing most of the required changes, but as the previous reviewer said [here (https://github.com/BenM4k/First_Capstone_Project/pull/1#discussion_r1171221722) on the fourth point that you should add an hover effect to the card, that is what the whiteness around one of those cards indicate, Kindly implement that 😉.

AdnanOlarmmi commented on pages/about.html:
```diff
@@ -0,0 +1,105 @@
+        <div class="menu-btn">
+            <span class="burger"></span>
+        </div>
+        <ul class="sm">
+            <li><a href="facebook.com" class="fa fa-facebook smicon"></a></li>
+            <li><a href="twitter.com" class="fa fa-twitter smicon"></a></li>
+            <li><a href="#" class="smicon">English</a></li>
+            <li><a href="#" class="smicon">My page</a></li>
+            <li><a href="#" class="smicon">Logout</a></li>
+        </ul>
```
- [x] Kindly remove the social media icons and those things from the about page's mobile version, 😃.

AdnanOlarmmi reviewed (changes requested):
Hi @BenM4k, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

![You're Almost there, funny one eyed alien](https://user-images.githubusercontent.com/41163571/217957948-6d8d9f33-125c-4724-954f-330eed60533c.gif)

## Highlights
- Nice one fixing most of the required changes.
- Responsive design.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenM4k committed:
fix: html

fixed images locations

BenM4k committed:
fix: css

fixed location error

BenM4k committed:
fix: css

fixing location problems

BenM4k committed:
fix: css

fixed linter errors

BenM4k committed:
fix: css

fixed location errors

OyePriscilla commented on sass/_programs.scss:
```diff
@@ -0,0 +1,65 @@
+        font-size: 12px;
+      }
+
+      &:hover {
+        background-color: rgba(170, 167, 167, 0.479);
```
- [ ] One more thing, can we please kindly design this according to the given template? The hover effect should place a border (color of #fff) around the selected card, I think that's what the previous reviewer meant by "whiteness around the cards".

## Given template
<img width="360" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/233301198-322c2df0-2fd5-43c8-ae1c-2363b6b46d3d.png">

OyePriscilla commented on js/about.js:
```diff
@@ -0,0 +1,23 @@
+// navbar
+const menuBtn = document.querySelector('.menu-btn');
+const burger = document.querySelector('.burger');
+const header = document.querySelector('header');
+const menuNav = document.querySelector('.menu-navbar');
+let displayMenu = false;
+
+function toggleMenu() {
+  if (!displayMenu) {
+    burger.classList.add('open');
+    header.classList.add('open');
+    header.classList.add('open');
+    menuNav.classList.add('open');
+    displayMenu = true;
+  } else {
+    burger.classList.remove('open');
+    header.classList.remove('open');
+    menuNav.classList.remove('open');
+    displayMenu = false;
+  }
+}
+
+menuBtn.addEventListener('click', toggleMenu);
```
- [optional] All these lines of code had already been implemented in `main.js`, I think you can remove it and just add `main.js` script to your `about.html` file. 👍  This will ensure that we follow DRY principle. Thanks. 👍

OyePriscilla commented on js/main.js:
```diff
@@ -0,0 +1,67 @@
+
+// feature speakers
+
+const speakers = [
+  {
+    id: '01',
+    name: 'Yochai Benkler',
+    title: 'Lorem ipsum dolor sit amet.',
+    paragraph: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, nisi esse.',
+    img: 'assets/img1.jpg',
+  },
+  {
+    id: '02',
+    name: 'Yochai Benkler',
+    title: 'Lorem ipsum dolor sit amet.',
+    paragraph: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, nisi esse.',
+    img: 'assets/img2.jpg',
+  },
+];
```
- [Optional] For your project to be more robust and engaging, I suggest you add more speakers to this section. ℹ️

binhussen replied:
- [Optional] Kindly, It would be better if you add more speakers to make it more engaging and stick to the given design as the previous reviewer suggested you 👌

OyePriscilla reviewed (changes requested):
Hi @BenM4k 👏 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so-close-so-close-seth-meyers](https://user-images.githubusercontent.com/103386183/196425724-8ca5825c-b02a-4983-b976-4bd0a2f70f9a.gif)

 ## Highlights
Use Flexbox to place elements in the page ✔️
Use GitHub Pages to deploy web pages ✔️
Use JavaScript to manipulate DOM elements ✔️
Use JavaScript events ✔️
Use objects to store and access data ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenM4k committed:
fix: css

changed the hover effect

binhussen reviewed (approved):
Hi @BenM4k , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
<img src="https://user-images.githubusercontent.com/65852621/133696182-6c5b784b-0083-427a-b24d-3ec870915f79.jpg"/>
## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
